velox/core/QueryConfig.h

```diff
@@ -389,6 +398,27 @@ class QueryConfig {
     return get<bool>(kOperatorTrackCpuUsage, true);
   }
 
+  uint32_t taskWriterCount() const {
+    const auto writerCount = get<uint32_t>(kTaskWriterCount, 1);
+    VELOX_CHECK(
+        bits::isPowerOfTwo(writerCount),
+        "'task_writer_count' must be power of 2, the actual is {}",
```

Comment:
Curious, where does this limitation come from?

Reply:
This is from java.

Reply:
This looks like a weird limitation to me. Would be nice to avoid it if possible. Otherwise, let's document it.